backend/src/main/java/ca/bc/gov/hlth/mohums/webclient/WebClientService.java

```diff
@@ -51,31 +52,29 @@ public WebClientService(WebClient kcAuthorizedWebClient) {
         return post(usersPath, data);
```

Comment:
Notably the `post` still returns `Mono<ClientResponse>`. But it works for the purpose the HTTP status codes so I'll do it later if at all.

Reply:
I think we're about to be adding a bunch more posts, and puts (user attributes, roles, and groups), should we just implement this now?